### DISCUSSION TITLE
Publish docker image for multiple PHP Versions

### DISCUSSION THREAD

**Discussion by fkrauthan** [php/frankenphp#78] [Q&A]
I was wondering if there are any plans to publish individual docker tags for different PHP versions. E.g. `dunglas/frankenphp:8.0-alpine` so that I would automatic get the latest 8.0 base image with the latest frankenphp server ontop.

**Comment by withinboredom** [comment]
I believe FrankenPHP only supports PHP 8.2 or higher, but once 8.3/9.0 comes out, I'm sure it will be like that.

**Reply by fkrauthan** [reply]
Ah that was my next question as I saw it currently has 8.2-rc as version. But might be good to start that tagging behaviour already so people can target 8.2 explicit and not run into issues when 9.0 comes out and they forget to change there base tag.

**Reply by withinboredom** [reply]
That's a pretty good point and worth doing.

**Comment by dunglas** [comment]
At some point, we'll probably have to create a GitHub Action bot similar to the one used for official images to generate new Docker images automatically when a new PHP version is published.
